### ISSUE TITLE
Update electrical properties

### ISSUE THREAD

**Issue by jankaeh** [openego/osmTGmod#15]
I opened a pull request some time ago, adapting corrections made at wupperinst/osmTGmod:
https://github.com/openego/osmTGmod/pull/14
The ohmic resistance of 110 and 220 kV lines was to high and s_nom of 380/220 kV trafos was to high.
If I understand this right, the properties of the lines can be changed without problems as post processing. But as the number of transformers placed at a bus depend on the capacity of the connected lines and the standard capacity of the transformers, this is not possible for the transformers without increasing their number.
So for a future run of an abstraction it might be nice to include this.

**Comment by ulfmueller** [comment]
Yes you are totally right. @maltesc just recently pointed this out as well.

We are currently mainly using lopf models on the data model so the resistance error did/does not bother us. But of course it is important to tackle that error. 

Unfortunately due to a time lack we will not create a new abstraction in the following month so it will probably for now be a post processing. But like you said once there is a new run, this bug will be gone. I added @maltesc and @lukasoldi as reviewer for your PR. Thx!
